via/models/video.py

```diff
@@ -0,0 +1,11 @@
+from sqlalchemy.orm import Mapped, mapped_column
+
+from via.db import Base
+from via.models._mixins import AutoincrementingIntegerIDMixin, CreatedUpdatedMixin
+
+
+class Video(AutoincrementingIntegerIDMixin, CreatedUpdatedMixin, Base):
+    __tablename__ = "video"
+
+    video_id: Mapped[str] = mapped_column(unique=True)
```

Comment:
Now that we already have transcripts in the DB it's probably too complicated to be worth it but we could have a FK in transcripts pointing to this table.

We can join now by video_id if we need to, it has an index in both tables.

Reply:
Yeah, I thought about this but I didn't really think it was worth the complexity. We don't need an fkey in order to look up a `Video` given a `Transcript`, as you said. So I'm not sure an fkey would really buy us anything. In fact, I think we might actually _not_ want to enforce that every `Video` in the DB must have a matching `Transcript` or vice-versa, the order of operations (API calls etc) may call for the `Video` alone to first be saved and then the transcript not downloaded until a later request or vice-versa. I suppose this does mean that any code that has a `Video` cannot assume that a matching `Transcript` exists or vice-versa, which may lead to a little more code complexity at some point.